package.json

```diff
@@ -40,13 +40,13 @@
   "dependencies": {
     "body-parser": "^1.6.5",
     "express": "4.x",
-    "grunt-mocha": "0.4.x",
+    "grunt-lib-phantomjs": "^0.7.0",
     "istanbul": "^0.3.0",
     "jade": "1.x",
     "lodash": "^2.4.1",
-    "moment": "^2.8.1",
     "minimatch": "^2.0.4",
-    "phantomjs": "1.9.13",
```

Comment:
Ug.  I just realized that this PR includes a newer version of grunt-lib-phantomjs, which pulls in phantomjs-1.9.17, rendering this line in package.json useless.